docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
It allows import of both types and functions. Without it we need to import everything separately. Since the migration tool automatically does this and all examples use it, it seemed to be what users are most likely to be familiar with. We can incrementally switch to individual imports.